models/examples/example-1-repeat-dispensing/SendRequest-FhirMessageSigned.json

```diff
@@ -135,6 +135,18 @@
             "valueReference": {
               "reference": "urn:uuid:DBAA616B-E23C-4D53-0F82-B2D178DB79CA"
             }
+          },
+          {
```

Comment:
this would actually help clean up the test files a bit - it would show that a MedicationRequest can still be processed without PrescriberEndorsements

Reply:
as discussed, has impact on other areas of improvements we are considering making which could do with being defined first

Reply:
okay sure, lets keep it in mind after we decide how to move forward